### PR TITLE
Updated markdown draft explanation with example

### LINKED ISSUE #910: Basics>Markdown: Markdown Drafts
# Problem

[Relative Path](https://github.com/withastro/docs/blob/main/src/pages/en/guides/markdown-content.md)

## Issue 1:
These section is a bit confusing to [read](https://docs.astro.build/en/guides/markdown-content/#markdown-drafts).
image

## Issue 2:

I have read over this several times and absolutely can't grasp what this is trying to get across. 

image

As a new user of the framework I keep seeing Astro.glob(), but no references to it. Which could be linked to [this](https://docs.astro.build/en/reference/api-reference/) page. The other reference I found was in the [troubleshoot](https://docs.astro.build/en/guides/troubleshooting/#astroglob---no-matches-found).

However, both of these do not provided any clarity on how to filter `data` via the `Astro.glob()` as this sentence suggests. 

>Aside: I just realized that the "filters to exclude any draft posts" is bold. It's not easily seen. Possible to set a CSS styling to these?

### PR DESCRIPTION
**Pull request by sarah11918**
- New or updated content
- Closes #910 

Simplifies wording for Markdown drafts, and provides a short example of filtering as well as a link back to `Astro.glob()`

### CODE REVIEW

File: src/pages/en/guides/markdown-content.md [hunk 1]
```diff
@@ -142,10 +142,14 @@ To build and publish this post:
 ```
 
 :::caution[Drafts and Astro.glob()]
-Although `draft: true` will prevent a page from being built on your site at that page route, `Astro.glob()` currently returns **all your Markdown files**.
+Although `draft: true` will prevent a page from being built on your site at that page route, [`Astro.glob()`](https://docs.astro.build/en/reference/api-reference/#astroglob) currently returns **all your Markdown files**.
```
Comment (sarah11918) [inline src/pages/en/guides/markdown-content.md:145]:
```suggestion
Although `draft: true` will prevent a page from being built on your site at that page route, [`Astro.glob()`](/en/reference/api-reference/#astroglob) currently returns **all your Markdown files**.
```

File: src/pages/en/guides/markdown-content.md [hunk 2]
```diff
@@ -142,10 +142,14 @@ To build and publish this post:
...
-Although `draft: true` will prevent a page from being built on your site at that page route, `Astro.glob()` currently returns **all your Markdown files**.
+Although `draft: true` will prevent a page from being built on your site at that page route, [`Astro.glob()`](/en/reference/api-reference/#astroglob) currently returns **all your Markdown files**.
 :::
 
-To exclude the data (e.g. title, link, description) from a draft post from being included in your post archive, or list of most recent posts, be sure that your `Astro.glob()` function also **filters to exclude any draft posts**.
+To exclude draft posts from being included in a post archive, or list of most recent posts, you can filter the results returned by your `Astro.glob()`.
+
+```
+const posts = await Astro.glob('../pages/post/*.md').filter((post) => !post.frontmatter.draft);
+```
```
Comment (delucis) [inline src/pages/en/guides/markdown-content.md:152]: Excellent addition! Thank you 🚀 

I’d suggest adding `js` to the opening tag to get syntax highlighting and also split this across two lines so readers don’t have to scroll horizontally to see the full example:

````suggestion
```js
const posts = await Astro.glob('../pages/post/*.md')
  .filter((post) => !post.frontmatter.draft);
```
````

**Review by delucis — APPROVED** [review]
Perfect. LGTM!

**Comment by Hopelezz** [comment]
#TeamAstro doing great work!

**Pull request merged**